pkg/index/validate.go

```diff
@@ -74,6 +75,11 @@ func (p Plugin) Validate(name string) error {
 			return errors.Wrapf(err, "platform (%+v) is badly constructed", pl)
 		}
 	}
+	if p.Spec.Homepage != "" {
```

Comment:
don't validate this for now.

1. we don't get anything out of it
2. ParseRequestURI is **only** meant to be used for parsing URLs that came in a request (not user input).

Reply:
If we want the URL to be used in the homepage field, I think we should better validate the value. But I don't disagree strongly, so I'll remove validating code.

Reply:
Does url.Parse give enough to you?

Reply:
Yes, it does.

Reply:
Not related to this PR, but the `shortDescription` field seems that does not be contained in the output fields in spite of required field.

Reply:
> Does url.Parse give enough to you?

I'm sorry, actualy not enough :bow:. Because `url.Parse` does not return error with value `this-is-not-a-valid-url`, so I think this validating code has not meaning.

 I removed the code by 32c6d76.